docs/sphinx-docs/source/_extensions/dollarmath.py

```diff
@@ -37,9 +37,9 @@ def test_dollar():
     assert replace_dollar(u"so is $last$")==u"so is :math:`last`"
     assert replace_dollar(u"and $mid$ too")==u"and :math:`mid` too"
     assert replace_dollar(u"$first$, $mid$, $last$")==u":math:`first`, :math:`mid`, :math:`last`"
-    assert replace_dollar(u"dollar\$ escape")==u"dollar$ escape"
-    assert replace_dollar(u"dollar \$escape\$ too")==u"dollar $escape$ too"
-    assert replace_dollar(u"emb\ $ed$\ ed")==u"emb\ :math:`ed`\ ed"
+    assert replace_dollar(r"dollar\$ escape")==u"dollar$ escape"
+    assert replace_dollar(r"dollar \$escape\$ too")==u"dollar $escape$ too"
+    assert replace_dollar(r"emb\ $ed$\ ed")==r"emb\ :math:`ed`\ ed"
```

Comment:
These changes contradict the changes made in #3560. Which change is the correct one?

Reply:
I can now confirm that these lines read as I expect, and outlined above.